src/library/required/extension/omg-pixel-collector-web-preloader.js

```diff
@@ -36,7 +36,7 @@
     omg.udo = {
         logFlattenedUdo: function () {
             var tagLoggingConfig = {
-                "stream": !omg.isProd(),
+                "stream": omg.isProd(),
```

Comment:
wouldn't we want it to just be true?

Reply:
depends, do we also want it for non-prod?

Reply:
considering that it was already on for non-prod, probably. want to keep the logging in both environments the same.